### PR TITLE
Refactoring live edit

### PR DESCRIPTION
**Pull request by Hegazy360**

### CODE REVIEW

File: addon/components/hyper-table/cell-renderers/date.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export default Component.extend({
       this.set("isEditing", false);
     },
     updateDate(date) {
-      this.manager.updateColumnValue(this.column.property, this.item, +date[0]/1000);
+      this.item.set(this.column.property, +date[0]/1000);
```
Comment (phndiaye) [inline addon/components/hyper-table/cell-renderers/date.js]: why ??

File: addon/mixins/editable.js [hunk 2]
```diff
@@ -19,16 +19,12 @@ export default Mixin.create({
...
-          this.manager.updateColumnValue(this.column.key, this.item, value);
-          this.manager.hooks.onLiveEdit({key: this.column.key, field: this.item, value});
-        });
+        this.manager.hooks.onLiveEdit({key: this.column.key, field: this.item, value});
       }
     },
 
     updateCurrency(currency) {
-      this.manager.updateColumnValue(this.column.currency_key, this.item, currency);
+      this.item.set(this.column.currency_key, currency);
```
Comment (phndiaye) [inline addon/mixins/editable.js]: why ??

**Comment by phndiaye** [comment]
LGTM 👍

File: addon/components/hyper-table/cell-renderers/date.js (hunk 1 above)
Comment (Hegazy360) [reply addon/components/hyper-table/cell-renderers/date.js]: simplifying, updateColumnValue did the same thing `item.set(key, value);`

**Pull request merged**